README.md

```diff
@@ -1,262 +1,87 @@
-# Deno Starter Template for Run On Slack apps
+# Deno Template App
 
-This repo is a starter template for new Run On Slack apps. This template uses Deno and the Slack CLI.
+This is a generic Deno template app used to build out Slack apps using the Slack
+CLI.
 
-Before getting started, make sure you have a development workspace where you have permissions to install apps. If you don’t have one set up, go ahead and [create one](https://slack.com/create).
+Before getting started, make sure you have a development workspace where you
+have permissions to install apps. If you don’t have one set up, go ahead and
+[create one](https://slack.com/create).
 
-## Installation
+## Setup
 
-#### Prerequisites
+### Install the Slack CLI
 
-To use this template, you will need to have installed and configured the Slack CLI. 
-Do this by following our [Quickstart Guide](https://api.slack.com/future/quickstart).
+To use this template, you first need install and configure the Slack CLI.
+Step-by-step instructions can be found in our
+[Quickstart Guide](https://api.slack.com/future/quickstart).
 
-#### Get Started
+## Running Your Project Locally
 
-Make a Run On Slack app with this with repo by **creating an app from this template**, **configuring your app**, then **writing functions**. 
-
-Once you're done, you can run your app with the CLI's local development server or Deploy your app to production.
-
-### Setup Your Project
+While building your app, you can see your changes propagated to your workspace
+in real-time with `slack run`. You'll know an app is the development version
+because the name has the string `(dev)` appended.
 
 ```zsh
 # Clone this project onto your machine
-slack create my-app -t slack-samples/deno-starter-template
+$ slack create my-app -t slack-samples/deno-starter-template
 
 # Change into this project directory
-cd my-app
+$ cd my-app
 
 # Run app locally
-slack run
-
-# Deploy app
-slack deploy
-```
-
-#### Configuring Your App
```

Comment:
A few reasons for the removal of a lot of this:

1. This template used to be blank, and so extensive instructions and explanations made sense. However, with the project changes, that information is no longer necessary: those structures are in place already.
2. As quickly as things change, having content embedded within the README is going to become untenable. We should be linking out to documentation as much as possible, which this document now does.

Reply:
Totally makes sense. I think there are some gaps between what's current on the api docs site and what's being removed here (not that we shouldn't move forward to get this info shifted over to the api docs site). In the event that the existing Readme was useful in building out those api docs pages we can share a link to the prior version of the readme before we merged this removal in.